From: Adam Conrad <adconrad@ubuntu.com>
Date: Wed, 25 May 2022 18:22:00 +0800
Subject: Remove resolvconf artifacts

 If chroot/etc/resolvconf/resolv.conf.d exists, tidy up the original and
 tail files, which are artifacts of the build (and leak the host system's
 resolv.conf).
Last-Update: 2012-08-20
---
 scripts/build/lb_chroot_resolv | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/scripts/build/lb_chroot_resolv b/scripts/build/lb_chroot_resolv
index 417c412..37bf8af 100755
--- a/scripts/build/lb_chroot_resolv
+++ b/scripts/build/lb_chroot_resolv
@@ -90,6 +90,12 @@ case "${1}" in
 			Truncate chroot/etc/resolv.conf
 		fi
 
+		# Clean up resolvconf's pollution
+		if [ -d chroot/etc/resolvconf/resolv.conf.d ]; then
+			rm -f chroot/etc/resolvconf/resolv.conf.d/original
+			rm -f chroot/etc/resolvconf/resolv.conf.d/tail
+		fi
+
 		# Removing stage file
 		rm -f .build/chroot_resolv
 		;;
